monai/engines/utils.py

```diff
@@ -75,16 +75,29 @@ def get_devices_spec(devices: Optional[Sequence[torch.device]] = None) -> List[t
 
 
 def default_prepare_batch(
-    batchdata: Dict[str, torch.Tensor]
-) -> Union[Tuple[torch.Tensor, Optional[torch.Tensor]], torch.Tensor]:
+    batchdata: Dict[str, torch.Tensor],
+    device: Optional[Union[str, torch.device]] = None,
```

Comment:
there's a `non_blocking: bool = False` in ignite, why not add the option here as well?

Reply:
Sure, added `non_blocking` support to all the engines, to be consistent with the latest ignite.
Thanks.